chat-components/database/src/main/java/com/pubnub/components/data/message/CustomContentConverter.kt

```diff
@@ -2,18 +2,26 @@ package com.pubnub.components.data.message
 
 import androidx.room.TypeConverter
 import com.google.gson.GsonBuilder
-import com.google.gson.reflect.TypeToken
 
 class CustomContentConverter {
-    private val gson = GsonBuilder()
-        .enableComplexMapKeySerialization()
-        .create()
 
     @TypeConverter
```

Comment:
@kazydek Converter updated, we can prepare a sample code with the usage

Reply:
yup, as part of another task